src/features/groups/list/category-tab/index.styled.ts

```diff
@@ -0,0 +1,37 @@
+import styled, { css } from 'styled-components';
+
+export const CategoryTabWrapper = styled.div`
+  display: inline-flex;
+  flex-direction: column;
+`;
+
+interface ICategoryTabProps {
```

Comment:
Maybe it is better to call it FilterTabs? It is not really a categories.
We potentially could have categories like: traveling, home and so on. However, we won't use them as a filter.